django/db/migrations/executor.py

```diff
@@ -60,17 +64,32 @@ def migration_plan(self, targets):
     def migrate(self, targets, plan=None, fake=False):
         """
         Migrates the database up to the given targets.
+
+        Django first needs to create all project states before a migration is
+        (un)applied (#24123) and in a second step run all the database
```

Comment:
In general we try to avoid including ticket numbers in the non-test code unless there is some information in the ticket that is worth referencing that cannot easily be explained in a comment.